### DISCUSSION TITLE
clicking off windows dismisses window

### DISCUSSION THREAD

**Discussion by tonyguadagno** [modoboa/modoboa#1725] [Q&A]
hi, i am using Modoboa 1.13.1 installed with auto installer on CentOS7/Apache

i have noticed that if you are in the middle of a task (for example, creating a new account) and accidentally click on the background tab (instead of the create account window) it dismisses the window and treats it like a cancel.
  In my opinion, it should just ignore that click and let you proceed.
![image](https://user-images.githubusercontent.com/25204879/56744149-f27a0c80-6745-11e9-8751-f64f5875b7d7.png)

**Comment by mertowitch** [comment]
Why does we need this?

**Comment by tonyguadagno** [comment]
Hi, I am not sure I understand your comment, but I feel it is important because I often accidentally click short of the "next" button, or tap my touch pad, which causes me to loose my work.  I think it makes more send (to me at least) for the behavior to be "ignore" rather than "cancel"
